folly/io/test/CompressionTest.cpp

```diff
@@ -156,6 +169,10 @@ class CompressionTest
 };
 
 void CompressionTest::runSimpleIOBufTest(const DataHolder& dh) {
+  if (!codec_) {
+    return;
```

Comment:
This looks like falsely-passing tests. It would be better if these tests were never run. This can be accomplished by computing the list of codec types to instantiate test cases for. I think you can use a `vector` returned from some computation as an argument to `INSTANTIATE_TEST_CASE_P`.